### ISSUE TITLE
[Descriptor Issue]: c3d operations

### ISSUE THREAD

**Issue by kaitj** [childmindresearch/niwrap#155]
### Package

Convert3D

### Tool Name

c3d 

### Problem Description

When calling `c3d` using certain operations, the compiled wrapper expects there to be an `outputs` attribute for all operations, while this is not the case.

Likely an underlying `styx` bug

### Steps to Reproduce

Call C3d with an operation that doesn't have an `output` attribute (eg. `C3dResampleMm`)

```python
    c3d.c3d_(
        input_=[...],
        operations=c3d.C3dResampleMm(res),
        output=...
     )
 ```

### Logs

```shell
Traceback (most recent call last):
  File "/local/.venv/bin/nhp_dwiproc", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/local/.venv/lib/python3.11/site-packages/nhp_dwiproc/run.py", line 23, in main
    app.analysis_levels.preprocess.run(cfg=cfg, logger=logger)
  File "/local/.venv/lib/python3.11/site-packages/nhp_dwiproc/app/analysis_levels/participant/preprocess.py", line 124, in run
    ref_b0, transforms = preprocess.registration.register(
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/local/.venv/lib/python3.11/site-packages/nhp_dwiproc/workflow/diffusion/preprocess/registration.py", line 87, in register
    ref_b0 = c3d.c3d_(
             ^^^^^^^^^
  File "/local/.venv/lib/python3.11/site-packages/niwrap/c3d/c3d.py", line 8828, in c3d_
    operations=operations.outputs(execution),
               ^^^^^^^^^^^^^^^^^^
AttributeError: 'C3dResampleMm' object has no attribute 'outputs'
```

**Comment by nx10** [comment]
This should be fixed in the dev/github version. Can you confirm?

**Comment by kaitj** [comment]
Was fixed as of `0.1.3` I believe.
